search-app/src/cmr/search/services/query_service.clj

```diff
@@ -222,8 +222,8 @@
                                                                     concept-type
                                                                     query))
         total-took (+ query-creation-time find-concepts-time)]
-    (info (format "Found %d %ss in %d ms in format %s with JSON Query %s and query params %s."
-                  (:hits results) (name concept-type) total-took
+    (info (format "Found %d %ss in %d ms from client %s in format %s with JSON Query %s and query params %s."
```

Comment:
Nitpick: have each argument on its own line for each of these calls

```
(info (format "Found %d %ss in %d ms from client %s in format %s with JSON Query %s and query params %s."                 
              (:hits results)
              (name concept-type)
              total-took
              (:client-id context)
              (rfh/printable-result-format (:result-format query)) json-query (pr-str params)))
```

Reply:
I did that refactor and then took it back - notice you have 3 more on the last line that should also be put on their own line. Lots of extra whitespace and didn't seem to improve readability (times the 5 places to do this in the file).